isos/bootstrap/bootstrap

```diff
@@ -108,7 +108,7 @@ if timeout -s KILL 2m mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}
     # mount the cgroup hierarchy to allow DinV
     mount -t cgroup -o all cgroup /sys/fs/cgroup
 
-    until [[ $(ls -1 /dev/disk/by-label | wc -l) -eq $(ls -1 --ignore='loop*' --ignore='ram*' /sys/block | wc -l) ]]; do sleep 0.1;done
+    until [[ $(ls -1 /dev/disk/by-label | wc -l) -eq $(ls -l /sys/block/ | grep -v 'loop*' | grep -v 'ram*' | grep -v 'total' | wc -l) ]]; do sleep 0.1;done
```

Comment:
Damn it - managed to pick that up from a different stashed change. Also, that's supposed to be a `1 (one)` not an `l (ell)`.
```suggestion
    until [[ $(ls -1 /dev/disk/by-label | wc -l) -eq $(ls -1 /sys/block/ | grep -v 'loop*' | grep -v 'ram*' | wc -l) ]]; do sleep 0.1;done
```